cs-config/cs_config/helpers.py

```diff
@@ -8,6 +8,7 @@
 import gzip
 import pandas as pd
 from taxcalc import Policy
+from ogusa.constants import TC_LAST_YEAR
```

Comment:
Here I'm importing `TC_LAST_YEAR` and replacing all references to 2028 and 2029 with the value of this constant.  But I'm still finding issues with parameter validation when I set the max year to 2029.  @hdoupe

Reply:
@jdebacker `TC_LAST_YEAR` should be set to 2030 (or `taxcalc.Policy.LAST_BUDGET_YEAR`) here:
https://github.com/PSLmodels/OG-USA/blob/a0b968720115cc2533cac3ae3f9edb4d9a98d35c/ogusa/constants.py#L13

Reply:
@hdoupe Ok, I've made this change and also just pulled the last year from `Policy` in `helpers.py`.  When I run `pytest ` though, the reading and validation of imports fails:

```
=================================== FAILURES ===================================
________________________ TestFunctions1.test_get_inputs ________________________

self = <cs_config.tests.test_functions.TestFunctions1 object at 0x122dc90b8>

    def test_get_inputs(self):
        self.test_all_data_specified()
>       inputs = self.get_inputs({})

../../../../anaconda3/envs/ogusa-dev/lib/python3.6/site-packages/cs_kit/validate.py:95: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
functions.py:89: in get_inputs
    iit_params = TCParams()
../../../../anaconda3/envs/ogusa-dev/lib/python3.6/site-packages/paramtools/parameters.py:56: in __init__
    ) = schemafactory.schemas()
../../../../anaconda3/envs/ogusa-dev/lib/python3.6/site-packages/paramtools/schema_factory.py:94: in schemas
    defaults_schema.load(self.defaults),
../../../../anaconda3/envs/ogusa-dev/lib/python3.6/site-packages/marshmallow/schema.py:723: in load
    data, many=many, partial=partial, unknown=unknown, postprocess=True
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <DefaultsSchema(many=False)>
data = {'ACTC_ChildNum': {'description': 'Families with this number of qualified children or more may qualify for a different...axpayers.', 'notes': '', 'section_1': 'Child/Dependent Credits', 'section_2': 'Additional Child Tax Credit', ...}, ...}

    def _do_load(
        self,
        data: typing.Union[
            typing.Mapping[str, typing.Any],
            typing.Iterable[typing.Mapping[str, typing.Any]],
        ],
        *,
        many: bool = None,
        partial: typing.Union[bool, types.StrSequenceOrSet] = None,
        unknown: str = None,
        postprocess: bool = True
    ):
        """Deserialize `data`, returning the deserialized result.
        This method is private API.
    
        :param data: The data to deserialize.
        :param many: Whether to deserialize `data` as a collection. If `None`, the
            value for `self.many` is used.
        :param partial: Whether to validate required fields. If its
            value is an iterable, only fields listed in that iterable will be
            ignored will be allowed missing. If `True`, all fields will be allowed missing.
            If `None`, the value for `self.partial` is used.
        :param unknown: Whether to exclude, include, or raise an error for unknown
            fields in the data. Use `EXCLUDE`, `INCLUDE` or `RAISE`.
            If `None`, the value for `self.unknown` is used.
        :param postprocess: Whether to run post_load methods..
        :return: Deserialized data
        """
        error_store = ErrorStore()
        errors = {}  # type: typing.Dict[str, typing.List[str]]
        many = self.many if many is None else bool(many)
        unknown = unknown or self.unknown
        if partial is None:
            partial = self.partial
        # Run preprocessors
        if self._has_processors(PRE_LOAD):
            try:
                processed_data = self._invoke_load_processors(
                    PRE_LOAD, data, many=many, original_data=data, partial=partial
                )
            except ValidationError as err:
                errors = err.normalized_messages()
                result = (
                    None
                )  # type: typing.Optional[typing.Union[typing.List, typing.Dict]]
        else:
            processed_data = data
        if not errors:
            # Deserialize data
            result = self._deserialize(
                processed_data,
                error_store=error_store,
                many=many,
                partial=partial,
                unknown=unknown,
            )
            # Run field-level validation
            self._invoke_field_validators(
                error_store=error_store, data=result, many=many
            )
            # Run schema-level validation
            if self._has_processors(VALIDATES_SCHEMA):
                field_errors = bool(error_store.errors)
                self._invoke_schema_validators(
                    error_store=error_store,
                    pass_many=True,
                    data=result,
                    original_data=data,
                    many=many,
                    partial=partial,
                    field_errors=field_errors,
                )
                self._invoke_schema_validators(
                    error_store=error_store,
                    pass_many=False,
                    data=result,
                    original_data=data,
                    many=many,
                    partial=partial,
                    field_errors=field_errors,
                )
            errors = error_store.errors
            # Run post processors
            if not errors and postprocess and self._has_processors(POST_LOAD):
                try:
                    result = self._invoke_load_processors(
                        POST_LOAD,
                        result,
                        many=many,
                        original_data=data,
                        partial=partial,
                    )
                except ValidationError as err:
                    errors = err.normalized_messages()
        if errors:
            exc = ValidationError(errors, data=data, valid_data=result)
            self.handle_error(exc, data, many=many, partial=partial)
>           raise exc
E           marshmallow.exceptions.ValidationError: {'CPI_offset': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'FICA_ss_trt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'SS_Earnings_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'SS_Earnings_thd': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'FICA_mc_trt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMEDT_ec': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMEDT_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'SS_thd50': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'SS_percentage1': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'SS_thd85': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'SS_percentage2': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_StudentLoan_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_SelfEmploymentTax_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_SelfEmp_HealthIns_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_KEOGH_SEP_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_EarlyWithdraw_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_AlimonyPaid_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_AlimonyReceived_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_EducatorExpenses_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_HSADeduction_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_IRAContributions_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_DomesticProduction_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_Tuition_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_InvInc_ec_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_Dependents_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_Dependents_Child_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_Dependents_Elder_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_Dependents_thd': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ALD_BusinessLosses_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_em': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_em_ps': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_prt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_no_em_nu18': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'STD': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'STD_Dep': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'STD_Aged': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'STD_allow_charity_ded_nonitemizers': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_credit': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_credit_ps': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_credit_prt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_credit_nr': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_credit_nr_ps': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_credit_nr_prt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Medical_frt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Medical_frt_add4aged': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Medical_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Medical_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_StateLocalTax_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_StateLocalTax_crt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_StateLocalTax_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_RealEstate_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_RealEstate_crt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_RealEstate_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_AllTaxes_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_AllTaxes_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_InterestPaid_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_InterestPaid_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Charity_crt_all': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Charity_crt_noncash': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Charity_frt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Charity_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Charity_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Charity_f': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Casualty_frt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Casualty_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Casualty_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Miscellaneous_frt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Miscellaneous_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_Miscellaneous_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_ps': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_prt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_crt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_BenefitSurtax_trt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_BenefitSurtax_crt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_BenefitSurtax_em': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_BenefitSurtax_Switch': {'value': {119: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 120: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 121: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 122: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 123: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 124: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 125: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_BenefitCap_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_BenefitCap_Switch': {'value': {119: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 120: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 121: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 122: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 123: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 124: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 125: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_c': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_AmountCap_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ID_AmountCap_Switch': {'value': {119: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 120: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 121: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 122: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 123: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 124: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 125: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_rt1': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_brk1': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_rt2': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_brk2': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_rt3': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_brk3': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_rt4': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_CG_rt1': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_CG_brk1': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_CG_rt2': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_CG_brk2': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_CG_rt3': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_CG_brk3': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_CG_rt4': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_nodiff': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_ec': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CG_reinvest_ec_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt1': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_brk1': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt2': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_brk2': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt3': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_brk3': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt4': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_brk4': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt5': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_brk5': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt6': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_brk6': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt7': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_brk7': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'II_rt8': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt1': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_brk1': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt2': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_brk2': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt3': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_brk3': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt4': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_brk4': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt5': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_brk5': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt6': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_brk6': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt7': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_brk7': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_rt8': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_EligibleRate_active': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_EligibleRate_passive': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_wages_active_income': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_top_stacking': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_qbid_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_qbid_taxinc_thd': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_qbid_taxinc_gap': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_qbid_w2_wages_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_qbid_alt_w2_wages_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'PT_qbid_alt_property_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_em': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_prt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_em_ps': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_child_em': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_child_em_c_age': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_rt1': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_brk1': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_rt2': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AMT_em_pe': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CDCC_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CDCC_ps': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CDCC_crt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_c_under5_bonus': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_ps': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_prt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ACTC_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ODC_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'NIIT_thd': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'NIIT_PT_taxed': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'NIIT_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_c': {'value': {68: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 69: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 70: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 71: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_rt': {'value': {68: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 69: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 70: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 71: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_basic_frac': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_prt': {'value': {68: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 69: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 70: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 71: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_ps': {'value': {68: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 69: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 70: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 71: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_ps_MarriedJ': {'value': {68: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 69: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 70: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 71: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_MinEligAge': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_MaxEligAge': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_InvestIncome_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_excess_InvestIncome_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_indiv': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'EITC_sep_filers_elig': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'RPTC_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'RPTC_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'LLC_Expense_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ETC_pe_Single': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ETC_pe_Married': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ACTC_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ACTC_rt_bonus_under5family': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ACTC_Income_thd': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'ACTC_ChildNum': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_c': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_c_under5_bonus': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_for_all': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_ps': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_prt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_refund_limited': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_refund_limit_payroll_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CTC_new_refund_limited_all_payroll': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'FST_AGI_trt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'FST_AGI_thd_lo': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'FST_AGI_thd_hi': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AGI_surtax_trt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'AGI_surtax_thd': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'LST': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'UBI_u18': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'UBI_1820': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'UBI_21': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'UBI_ecrt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_RetirementSavings_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_ForeignTax_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_ResidentialEnergy_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_GeneralBusiness_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_MinimumTax_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_AmOppRefundable_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_AmOppNonRefundable_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_SchR_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_OtherCredits_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_Education_hc': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_Charity_rt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_Charity_f': {'value': {85: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 86: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 87: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 88: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}, 89: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'CR_Charity_frt': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_ssi_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_housing_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_snap_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_tanf_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_vet_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_wic_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_mcare_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_mcaid_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_oasdi_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_ui_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}, 'BEN_other_repeal': {'value': {17: {'year': ['Input 2030 must be one of 2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029']}}}}

../../../../anaconda3/envs/ogusa-dev/lib/python3.6/site-packages/marshmallow/schema.py:904: ValidationError
```
Thoughts?

Reply:
Sorry about the not-very-helpful error message. This should be `TC_LAST_YEAR + 1` since `range` cuts off the last value:

https://github.com/jdebacker/dynamic/blob/51dbe4063da1cb9f69411e5b54e9cd33daa0c1b1/cs-config/cs_config/helpers.py#L21

Reply:
Ah - thanks! Obvious now!